### PR TITLE
Completed javascript-if.

### PR DESCRIPTION
**Pull request by stefancarrera**
js-if

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+function isOldEnoughToDrink(person) {
+  var drinkingAge = (person.age >= 21);
+  return drinkingAge;
+}
+
+function isOldEnoughToDrive(person) {
+  var drivingAge = (person.age >= 16);
+  return drivingAge;
```
Comment (scbowler) [inline javascript-if/main.js:34]: This should just always return false

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+function categorizeAcidity(pH) {
+  if ((pH < 7) && (pH >= 0)) {
+    return 'acid';
+  } if ((pH <= 14) && (pH >= 7)) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
```
Comment (scbowler) [inline javascript-if/main.js:50]: If exactly 7, it should return `'neutral'`

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+function isUnderFive(number) {
+  var underFive = (number < 5);
+  return underFive;
+}
+
+function isEven(number) {
+  var remainder = (number % 2);
+  var even = (remainder === 0);
+  return even;
```
Comment (scbowler) [inline javascript-if/main.js:19]: Try doing this with a single if statement

**Review by scbowler — CHANGES_REQUESTED** [review]
Couple changes needed

**Comment by stefancarrera** [comment]
js-if2

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**